modules/ROOT/pages/microsoft-sharepoint-2010-connector.adoc

```diff
@@ -204,169 +204,228 @@ The Claims-Based authentication scheme has the following parameters:
++
+** Modified: Modification date of the item
+** PermMask
+** ProgId
+** showshiddenversion
+** UniqueId
++
+* File Add
++
 Adds a file to a Document or Picture library.
-
-==== File Get Content
-
++
+* File Get Content
++
 Retrieves the content of a file.
-
-==== File Get Metadata
-
++
+* File Get Metadata
++
 Retrieves the metadata of a file.
-
-==== File Delete
-
++
+* File Delete
++
 Deletes a file from a Document or Picture library.
-
-==== File Check Out
-
++
+* File Check Out
++
 Checks out a file from a document library.
-
-==== File Undo Check Out
-
++
+* File Undo Check Out
++
 Reverts an existing checkout for a file.
-
-==== File Check In
-
++
+* File Check In
++
 Checks in a file to a document library.
-
-==== File Publish
-
++
+* File Publish
++
 Submits the file for content approval.
-
-==== File Unpublish
-
++
+* File Unpublish
++
 Removes a file from content approval or unpublish a major version.
-
-==== File Approve
-
++
+* File Approve
++
 Approves a file submitted for content approval.
-
-==== File Deny
-
++
+* File Deny
++
 Denies approval for a file that was submitted for content approval.
-
-==== File Copy To
-
++
+* File Copy To
++
 Copies the file to the destination URL.
-
-==== File Query
-
++
+* File Query
++
 Retrieves all files from a folder that matches the specified criteria.
-
++
 Additionally to the selected fields, the following fields are always returned:
++
+** Created: Creation date of the item
+** Created_x0020_Date
+** DocIcon
+** Editor: A user
+** FileLeafRef: Name of the folder
+** FileRef: Relative URL of the folder
+** FSObjType
+** Last_x0020_Modified
+** _Level
+** MetaInfo
+** _ModerationStatus
++
+Moderation Status of the file if it belongs to a library that has moderation enabled
++
+** Modified
++
+Modification date of the item
++
+** PermMask
+** ProgId
+** showshiddenversion
+** UniqueId
 
-* PermMask
-* _ModerationStatus: Moderation Status of the file if it belongs to a Library that has moderation enabled
-* Editor: A user
-* Created: Creation date of the item
-* Modified: Modification date of the item
-* owshiddenversion
-* MetaInfo
-* FSObjType
-* FileLeafRef: Name of the folder
-* FileRef: Relative URL of the folder
-* UniqueId
-* _Level
-* ProgId
-* Last_x0020_Modified
-* Created_x0020_Date
-* DocIcon
-
-=== Reference Objects
-
-In case a *SharepointListReference* or *SharepointListMultiValueReference* is selected to be returned by a query, the returned value of the field depends on the value of the *Retrieve full objects for reference fields* parameter:
-
-* Not checked: A summary object containing the reference object’s ID and the reference object list’s ID:
+== Reference Objects
+
+In case a *SharepointListReference* or *SharepointListMultiValueReference* is selected to be returned by a query, the returned value of the field depends on the value of the Retrieve full objects for reference fields parameter:
```

Comment:
Should the parameter name be formatted so that the sentence is easier to read?
```suggestion
In case *SharepointListReference* or *SharepointListMultiValueReference* is selected for return by a query, the returned value of the field depends on the value of the `Retrieve full objects for reference fields` parameter:
```